app/src/androidTest/java/com/android/sample/screen/SwipePageTest.kt

```diff
@@ -79,6 +83,15 @@ class SwipePageTest : TestCase() {
 
     `when`(mockNavigationActions.currentRoute()).thenReturn(Route.SWIPE)
     `when`(mockNavigationActions.navigateTo(Route.AUTH)).then {}
+
+    // Init the filter
+    recipesViewModel.updateTimeRange(0f, 100f)
+    recipesViewModel.updateTimeRange(5f, 99f)
+    recipesViewModel.updatePriceRange(0f, 100f)
+    recipesViewModel.updatePriceRange(5f, 52f)
```

Comment:
These constants could go in the file for the constants

Reply:
Oups my bad I didn't see that it was in the test file :)

Reply:
I see what you mean but for testing, I think it's okay keeping them here.